examples/GitHubLovesSparkDotNet/GitHubLovesSparkDotNet.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
oh do we plan to have one project per scenario? :)

Reply:
At the moment, per our discussion, yes. But I can also have Microsoft.Spark.Scenarios. Let me know what you prefer.